services/QuillLMS/app/assets/stylesheets/rollup_email.css

```diff
@@ -0,0 +1,96 @@
+.email-body {
+  margin: 0px;
+  width: 1030px;
```

Comment:
You might want to make these set widths into max-widths (and add a `width: 100%` rule) so that they look okay on phones.